packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/ListView/tests/index.test.js

```diff
@@ -34,1310 +53,145 @@ describe('Admin | containers | ListView', () => {
-        padding-bottom: 8px;
-        padding-left: 16px;
-        border-radius: 4px;
-        border-color: #4945ff;
-        border: 1px solid #4945ff;
-      }
-
-      .c15 {
-        font-size: 0.75rem;
-        line-height: 1.33;
-        font-weight: 600;
-        color: #ffffff;
-      }
-
-      .c10 {
-        -webkit-align-items: center;
-        -webkit-box-align: center;
-        -ms-flex-align: center;
-        align-items: center;
-        display: -webkit-inline-box;
-        display: -webkit-inline-flex;
-        display: -ms-inline-flexbox;
-        display: inline-flex;
-        -webkit-flex-direction: row;
-        -ms-flex-direction: row;
-        flex-direction: row;
-        gap: 8px;
-      }
-
-      .c13 {
-        -webkit-align-items: center;
-        -webkit-box-align: center;
-        -ms-flex-align: center;
-        align-items: center;
-        display: -webkit-box;
-        display: -webkit-flex;
-        display: -ms-flexbox;
-        display: flex;
-        -webkit-flex-direction: row;
-        -ms-flex-direction: row;
-        flex-direction: row;
-      }
-
-      .c11 {
-        position: relative;
-        outline: none;
-      }
-
-      .c11 > svg {
-        height: 12px;
-        width: 12px;
-      }
-
-      .c11 > svg > g,
-      .c11 > svg path {
-        fill: #ffffff;
-      }
-
-      .c11[aria-disabled='true'] {
-        pointer-events: none;
-      }
+    render(App);
 
-      .c11:after {
-        -webkit-transition-property: all;
-        transition-property: all;
-        -webkit-transition-duration: 0.2s;
-        transition-duration: 0.2s;
-        border-radius: 8px;
-        content: '';
-        position: absolute;
-        top: -4px;
-        bottom: -4px;
-        left: -4px;
-        right: -4px;
-        border: 2px solid transparent;
-      }
+    const loadingElement = await screen.findByText('Loading content.');
 
-      .c11:focus-visible {
-        outline: none;
-      }
+    expect(loadingElement).toBeInTheDocument();
 
-      .c11:focus-visible:after {
-        border-radius: 8px;
-        content: '';
-        position: absolute;
-        top: -5px;
-        bottom: -5px;
-        left: -5px;
-        right: -5px;
-        border: 2px solid #4945ff;
-      }
+    await waitForElementToBeRemoved(() => screen.getByText('Loading content.'));
 
-      .c12 {
-        -webkit-text-decoration: none;
-        text-decoration: none;
-      }
+    expect(screen.getByText('http:://strapi.io')).toBeInTheDocument();
+  });
 
-      .c12[aria-disabled='true'] {
-        border: 1px solid #dcdce4;
-        background: #eaeaef;
-      }
+  it('should show a loader when permissions are loading', () => {
+    useRBAC.mockImplementation(() => ({
+      isLoading: true,
+      allowedActions: { canUpdate: true, canCreate: true, canDelete: true },
+    }));
 
-      .c12[aria-disabled='true'] .c14 {
-        color: #666687;
-      }
+    useNotification.mockImplementation(() => jest.fn());
```

Comment:
I'm not sure why you keep mocking this? I would just use the NotificationProvider – mocks are a bit brittle imo :)